ansible/roles/riakcs/tasks/admin_user.yml

```diff
@@ -12,7 +12,7 @@
 
 # Note this can't be notify because we need it to restart immediately
 - name: Restart RiakCS
-  command: riak-cs restart
+  service: name=riak-cs state=started enabled=yes
```

Comment:
the reason there's a bunch of these is that you can run this playbook in portions using tags/defaults and i make no assumption up the status of the service (beyond the fact that it's installed)